src/inputType/PanInput.ts

```diff
@@ -266,8 +266,10 @@ export class PanInput implements IInputType {
 			]);
 		const prevent = offset.some(v => v !== 0);
 		if (prevent) {
-			event.srcEvent.preventDefault();
-			event.srcEvent.stopPropagation();
+			const srcEvent = event.srcEvent;
+
+			srcEvent.cancelable && srcEvent.preventDefault();
```

Comment:
@WoodNeck 

If cancelable is false, are you trying to stop all drag events? 


I don't think a `hold` and `change` event should happen.